pyaerocom/conftest.py

```diff
@@ -155,6 +160,12 @@ def loaded_nasa_ames_example():
     #fp = TESTDATADIR.joinpath(TEST_PATHS['nasa_ames_sc550aer'])
     return EbasNasaAmesFile(NASA_AMES_FILEPATHS['scatc_jfj'])
 
+@pytest.fixture(scope='session')
+def tempdir(tmpdir_factory):
+    """Temporary directory for dumping data shared between tests"""
+    tmpdir= tmpdir_factory.mktemp('data')
+    return tmpdir
```

Comment:
Maybe this should have scope='module' to make sure that tests from different modules using this fixture will interfere?

Reply:
Actually, the idea is to allow them to look into the same temp directory.

Reply:
From different test files?